src/VisualStudio/Core/Impl/SolutionExplorer/AnalyzersFolderItem/AnalyzersFolderItemProvider.cs

```diff
@@ -24,37 +19,24 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplore
     [Export(typeof(IAttachedCollectionSourceProvider))]
     [Name("AnalyzersFolderProvider")]
     [Order(Before = HierarchyItemsProviderNames.Contains)]
+    [AppliesToProject("(CSharp | VisualBasic) & !CPS")] // in the CPS case, the Analyzers folder is created by the project system
```

Comment:
Nit: Define a string constant given it is used more than once?

Reply:
I thought about it, but I wasn't sure anything would be really clearer than just seeing the literal expression. And even if somebody _did_ have to change this, I imagine they'd have to re audit all of the other ones to see if they still made sense, so this isn't really a good case where a constant means you can change it in one place and not have to worry about bugs you introduced.